packages/block-editor/src/components/block-list/block-list-item.native.js

```diff
@@ -89,6 +93,50 @@ export class BlockListItem extends Component {
 		];
 	}
 
+	componentDidUpdate() {
+		this.scrollToSelectedBlockIfItsNotVisible();
+	}
+
+	scrollToSelectedBlockIfItsNotVisible() {
+		const { isSelected } = this.props;
+
+		if ( this.blockRef.current ) {
```

Comment:
Just a nitpick: this could be changed to `if ( !this.blockRef.current ) return;` to reduce the number of levels of indentation.

Reply:
Thanks for this. I did it this way because that's the way how I wrote code in Kotlin in WPAndroid and the linter would throw an error if I did this without the braces 😅